docs/blog/posts/v0.51.0.md

```diff
@@ -141,4 +141,16 @@ Thanks to these little improvements, the datasource discovery feature is now mor
 
 ## Community Dashboards
 
+Fresh off our talk at KubeCon EU 2025 in London - [Limitless Possibilities, Consistent Design: Crafting Dashboards With Perses DAC](http://youtube.com/watch?v=7h70Olo5Uzk), given by two perses mantainers, Nicolas Takashi and Antoine Thébaud, we are happy to introduce the [community-dashboards](https://github.com/perses/community-dashboards) repository.
+
+Here we provide support for commonly used Mixin Dashboards, like Prometheus, Kubernetes and Thanos, build on top of **Dashboard as Code** capabilities provided by Perses. We’ll dive deeper into the community-dashboards in our upcoming blog post, but starting today, you can begin using them right away — whether by:
```

Comment:
Hi @Nexucis already referencing here the blog post. But I can change this a bit if it makes more sense

Reply:
oh sorry I miss read this line. All good for me then !